docs/components/TableRow.md

```diff
@@ -8,8 +8,8 @@ Props
 Prop                  | Type     | Default                   | Required | Description
 --------------------- | -------- | ------------------------- | -------- | -----------
 children|node||Yes|Contents of the row - should be cells.
-loading|bool|false|No|[PENDING DEPRECATION] renders the row as semitransparent.
 className|string|null|No|Adds a class name to the element.
 id|string|null|No|Adds an id to the element.
+onClick|func|() => null|No|Handler that is called whenever the row is clicked.
```

Comment:
Yeah, pretty much. Won't work well with links present within the row, but that's not the use case this table is meant to target.